db/db_impl/db_impl.cc

```diff
@@ -1496,6 +1496,29 @@ bool DBImpl::SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) {
   }
 }
 
+Status DBImpl::IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
```

Comment:
Also, it's not caused by this PR, but I think it would be nice to return an error, e.g. `Status::NotSupported()`, or `InvalidArgument()` if trying to increase/obtain `full_history_ts_low_` of a column family that disables timestamp.
Maybe add a check in this method and assert timestamp is enabled in `IncreaseFullHistoryTsLow(ColumnFamilyData*, std::string)` which can be renamed to something like`IncreaseFullHistoryTsLowImpl()`. The same for `GetFullHistoryTsLow()`.

Reply:
fixed